mintlify/remittances/developer-guides/implementation-overview.mdx

```diff
@@ -1,10 +1,6 @@
 ---
 title: "Implementation Overview"
-description: "An overview of the overall integration"
 ---
-
-# Introduction to UMA as a Service
-
 Welcome to the UMA as a Service (UMAaaS) documentation. UMA as a Service is an API that facilitates global payments to and from UMA addresses, which are human-readable addresses that can send to and from any currency.
```

Comment:
This seems like it needs to be tweaked along with a bunch of the UMAaaS mentions, etc.